### DISCUSSION TITLE
Model is not isible in IOS Safari but works fine on desktop one

### DISCUSSION THREAD

**Discussion by g-n-man** [google/model-viewer#4893] [Q&A]
Hello. Sorry for possibly a stupid question, but why does everything seem to work fine on the desktop but on IOS the model is not displayed?

https://github.com/g-n-man/test

**Comment by elalish** [comment]
It's working on my iOS devices. Also, "not displayed" is actually quite vague - some comparison screenshots would help, and is there anything in your JS console logs? Also, your link is to a Github repo, which I'm not about to pull. If you want to share repro steps, please link to a live website where we can see your problem.

**Reply by g-n-man** [reply]
Hello. Thank you for getting back to me. I am unsure how to check JS logs, but please find the link to a live website as well as screenshots from my laptop and phone below. I have also asked someone to test it on their iPad; it also works there (the model is visible and can be interacted with).

Website:
https://g-n-man.github.io/test/

Screenshots:
<img width="1352" alt="desktop" src="https://github.com/user-attachments/assets/aa0a37ee-81d7-4edf-bf6f-fe16540230ea">
![phone](https://github.com/user-attachments/assets/8ebf685a-69fa-4903-aded-d50b3d04ca92)

HTML code:
`<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8"/>
    <link rel="stylesheet" type="text/css" href="index.css"/>
    <style>
      #threedd {
        width: 100%;
        height: 80vh;
        margin-left: auto;
        margin-right: auto;
        background-color: rgb(210, 175, 175);
      }
    </style>
  </head>
  <body>
    <p>Hello</p>
    <model-viewer id="threedd" src="./3d_model_baked_comp.glb" camera-controls touch-action="pan-y"></model-viewer>

    <script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.5.0/model-viewer.min.js"></script>

  </body>
</html>`

**Reply by elalish** [reply]
What model of iPhone and what version of iOS? You can connect your phone to a Mac and use Safari's dev tools to connect to your phone's websites and inspect them to see JS logs, etc.

**Reply by g-n-man** [reply]
I have an iPhone 11 and iOS 17.6.1. 
Thank you for the suggestion,I'll try to do that
